src/contextmenu.js

```diff
@@ -21,12 +22,15 @@ function getVscodeLink({
     repo, file, isFolder, line,
 }) {
     return getOptions()
-        .then(({ insidersBuild, basePath, debug }) => {
+        .then(({ insidersBuild, remoteHost, basePath, debug }) => {
             let vscodeLink = insidersBuild
                 ? 'vscode-insiders'
                 : 'vscode';
 
-            vscodeLink += '://file';
+            // vscode://vscode-remote/ssh-remote+[host]/[path/to/file]:[line]
+            // OR
+            // vscode://file/[path/to/file]:[line]
+            vscodeLink += (remoteHost != '') ? '://vscode-remote/ssh-remote+' + remoteHost : '://file';
```

Comment:
Nitpicky but I think I'd find this slightly more readable

```suggestion
            if (remoteHost !== '') {
                vscodeLink += '://vscode-remote/ssh-remote+' + remoteHost;
            } else {
                vscodeLink += '://file';
            }
```